opam

```diff
@@ -20,7 +20,8 @@ remove: [["ocamlfind" "remove" "batteries"]]
 depends: [
   "ocamlfind" {>= "1.5.3"}
   "ocamlbuild" {build}
-  "qtest" {test & >= "2.0.0" & < "2.5"}
+  "qtest" {test & >= "2.0.0"}
+  "qcheck" {test & >= "0.6"}
```

Comment:
I find it suspicious that the qtest versions before and after the patch do not have an empty intersection. We used the `< 2.5` bound to not do the backward-incompatible change, you suggest to move to the new version, shouldn't we now require `>= 2.5`? Are the versions between `2.0` and `2.5` really compatible with the new code?